src/i18n.ts

```diff
@@ -0,0 +1,32 @@
+import * as i18n from 'i18next';
+import * as XHR from 'i18next-xhr-backend';
+import * as LanguageDetector from 'i18next-browser-languagedetector';
+
+const instance = i18n
+  .use(XHR)
+  .use(LanguageDetector)
+  .init({
+    fallbackLng: 'en',
+    debug: true,
+    interpolation: {
+      escapeValue: false, // not needed for react!!
+    },
+
+    // TODO: Look into loading js files into resources instead of json files
+    // resources: {},
```

Comment:
One thing I just thought about is we _could_ import all the different language translation resources and add them here. But unfortunately, we'd have one huge bundle with all translations. I guess it isn't really a big deal though considering gzip with heavily compress it.